Frameworks/CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -275,6 +275,10 @@ void _CFRuntimeBridgeTypeToClass(CFTypeID cf_typeID, const void *cls_ref) {
     __CFUnlock(&__CFBigRuntimeFunnel);
 }
 
+uintptr_t __CFISAForTypeID(CFTypeID typeID) {
```

Comment:
My proposal is to have it in the header for corefoundation-internal consumers, and have it forward-declared and exported for corefoundation-external consumers.

---
In reply to: [79034377](https://github.com/Microsoft/WinObjC/pull/1008#discussion_r79034377) [](ancestors = 79034377,79033450,79025442,79022634,78866010,78862337)